core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -5720,6 +5722,20 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
 
                     // i8x16 comparison operations
                     case SIMD_i8x16_eq:
+                    {
+                        V128 v1 = POP_V128();
+                        V128 v2 = POP_V128();
+                        int i;
+                        addr_ret = GET_OFFSET();
+
+                        V128 result;
+                        for (i = 0; i < 16; i++) {
```

Comment:
I think the comparison operations are fine, but this should have a proper SIMD implementation later.

Reply:
I already have this one in another branch, will raise it next